pkg/scaffold/internal/filesystem/filesystem_test.go

```diff
@@ -153,144 +151,6 @@ var _ = Describe("FileSystem", func() {
 		})
 	})
 
-	Describe("Errors", func() {
```

Comment:
HI @Adirio, 
 
My only concern here is: why these tests were removed? 
Could we not keep them? Otherwise, it shows ok

Reply:
They were moved to the `errors_test.go` file in the same directory.